src/eda_covid19.r

```diff
@@ -0,0 +1,88 @@
+# author: Fatime Selimi, Neel Phaterpekar, Nicholas Wu, Tanmay Sharma
+# date: 2020-11-25
+
+"Creates eda table and plots for the response ratio from the COVID-19 data (from https://github.com/owid/covid-19-data/tree/master/public/data).
+Saves the plots as a pdf and png file.
+Usage: src/eda_covid19.r --input_path=<input_path> --out_dir=<out_dir>
+  
+Options:
+--input_path=<input_path>     Path (including filename) to processed data in csv format
+--out_dir=<out_dir>           Path to directory where the plots should be saved
+" -> doc
+
+library(tidyverse)
+library(knitr)
+library(infer)
+library(cowplot)
+library(docopt)
+library(ggthemes)
+
+opt <- docopt(doc)
+
+main <- function(input_path, out_dir) {
+  
+    #create a table summarizing the key statistics
+    covid_CAN_USA <- read_csv(input_path)
+    
+    covid_CAN_USA_summary <-  covid_CAN_USA %>%
+        group_by(iso_code) %>%
+        summarise(sample_size = n(),
+                  mean_response_ratio = mean(response_ratio),
+                  median_response_ratio = median(response_ratio),
+                  standard_dev = sd(response_ratio)
+        )
+                         
+    #save table as a .png file
+    write_csv(covid_CAN_USA_summary, 
+              paste0(out_dir, "/covid_CAN_USA_summary.csv"))
+
+
+    #create the histogram plot
+    dist_CAN <- covid_CAN_USA %>%
```

Comment:
We can create a function out of this and then use it in both `dist_CAN` plot and `dist_USA` plot. No need to repeat the same code twice. 

E.g.
```
create_distribution_plot <- function(data, iso_code, country) {
    data %>%
    filter(iso_code == iso_code) %>%
    ggplot(aes(response_ratio)) +
    geom_histogram(bins=50) +
    xlab(paste0("Response Ratio, ", country)) +
    ggtitle(paste0("Sample Distribution of ", country, "'s Response Ratio"))
}
```
this function should live outside `main()` function and then to use it for `dist_CAN` and `dist_USA` we can do something like:
```
dist_CAN <- create_distribution_plot(covid_CAN_USA, "CAN", "Canada")
dist_USA <- create_distribution_plot(covid_CAN_USA, "USA", "USA")
```
I hope that makes sense.

Reply:
Make sense. Function created, thanks!